advanced-integration/v2/backend/dotnet/Server.cs

```diff
@@ -0,0 +1,191 @@
+using System;
+using System.IO;
+using System.Net.Http;
+using System.Text;
+using System.Threading.Tasks;
+using Microsoft.AspNetCore.Builder;
+using Microsoft.AspNetCore.Hosting;
+using Microsoft.AspNetCore.Http;
+using Microsoft.AspNetCore.Mvc;
+using Microsoft.Extensions.Configuration;
+using Microsoft.Extensions.DependencyInjection;
+using Microsoft.Extensions.Hosting;
+using Newtonsoft.Json;
+
+namespace PayPalAdvancedIntegration
```

Comment:
Is there a specific version of C#/.NET you're targeting here? I think it's probably more common to use a simple namespace declaration `namespace PayPalAdvancedIntegration;` instead of enclosing the whole file in curly braces.

Reply:
We are targeting v8.0. I'll remove the curly braces